src/app/contacts/contacts.component.ts

```diff
@@ -1,15 +1,18 @@
 import { Component, OnInit } from '@angular/core';
+import { ContactsInfo, ContactsService } from '../shared/services/contacts.service';
 
 @Component({
   selector: 'app-contacts',
   templateUrl: './contacts.component.html',
   styleUrls: ['./contacts.component.scss']
 })
 export class ContactsComponent implements OnInit {
+  contacts: ContactsInfo [];
 
-  constructor() { }
+  constructor( private info: ContactsService) { }
```

Comment:
@Yuriy-27 would be better using the same name to don't be confused in bigger components

like
`( private contactsService: ContactsService)`